client/my-sites/activity/filterbar/date-range-selector.jsx

```diff
@@ -64,7 +64,11 @@ export class DateRangeSelector extends Component {
 
 	handleDateRangeCommit = ( startDate, endDate ) => {
 		const { moment, selectDateRange } = this.props;
-		const formattedStartDate = startDate ? moment( startDate ).format( DATE_FORMAT ) : null;
+		const formattedStartDate = startDate
+			? moment( startDate )
+					.startOf( 'day' )
```

Comment:
```suggestion
					.startOf( 'day' )
					.utc()
```

Reply:
adding `utc()` is enough to make sure the correct dates are being sent, but it is essential that it is sent after `startOfDay()`!

Reply:
Good catch @cbauerman ! I guess I should add `utc()` to the end date as well, otherwise, I may see entries from the next day. 

I have pushed a new commit. Let me know what you think!